docs/docs/code/python/faqs/README.md

```diff
@@ -0,0 +1,25 @@
+# Frequently Asked Questions about Python
+
+## What's the difference between `def handler(pd)` and the `pipedream` package for Python code steps?
+
+The `handler` method is a newer implementation of Python code steps. It includes the ability to stop the workflow early, integrate a Data Store and/or Connected Accounts into your Python code step.
```

Comment:
Idea to simplify:

> The `pd` object passed to the `handler` method lets you exit the workflow early, integrate a Data Store, and use connected accounts into your Python code steps.

Reply:
I'd also recommend linking to the relevant sections of the docs for each of those three features listed.